tests/test_htmIndex.py

```diff
@@ -53,14 +54,15 @@ def make_coord(ra, dec):
 
 
 class HtmIndexTestCase(lsst.utils.tests.TestCase):
-
-    @staticmethod
-    def make_sky_catalog(out_path, size=1000):
+    @classmethod
```

Comment:
I don't see why this needs to be a classmethod. It looks like all of the setup happens in `setUpClass`, which is worrying because what happens if any of those values are changed by a test? I think the `setUpClass` should become just `setUp` and then this can just be a regular method.

Reply:
Much of the work happens in setupClass because it makes a butler, which is very expensive

Reply:
Good case for a `mock`? But not on this ticket, probably.